container/go/cmd/puller/puller.go

```diff
@@ -120,6 +122,56 @@ func pull(imgName, dstPath, format, cachePath string, platform v1.Platform) erro
 		}
 	}
 
+	if err := generateSym(img, dstPath); err != nil {
+		return errors.Wrapf(err, "failed to generate symbolic links to pulled image at %s", dstPath)
+	}
+
+	return nil
+}
+
+// generateSym creates symbolic links to the config.json and .tar.gz layers
+// for use with container_import rule.
+func generateSym(img v1.Image, dstPath string) error {
+	targetDir := path.Join(dstPath, "blobs/sha256")
+	symlinkDir := path.Join(filepath.Dir(dstPath), "image")
+
+	// symlink for config.json.
+	var config v1.Hash
+	var err error
+	if config, err = img.ConfigName(); err != nil {
+		return errors.Wrapf(err, "failed to get the config file's hash information for image")
+	}
+	configDir := path.Join(targetDir, config.Hex)
```

Comment:
Add a ospkg.Stat check for `configPath` to ensure the config file you are symlinking to actually exists. Do this for every symlink you create below as well.